### ISSUE TITLE
Updating Airsonic-advanced inside latest Docker Image

### ISSUE THREAD

**Issue by judicandus** [airsonic-advanced/airsonic-advanced#937]
EDIT: This was originally a post requesting a podcast feature that I learned had already been implemented in the latest version of airsonic-advanced. Turns out the latest docker image currently available (mid august 2022) has an old version of airsonic-advanced. So I changed this thread to how to update the airsonic inside an existing docker container. 

There is no possibility to change settings for individual podcasts. Some podcasts (like business daily) I don't want to keep all episodes but only the most recent ones, while other podcasts I want to keep all episodes and have them all downloaded. I couldn't find the option to change settings for individual podcasts but only global settings. 

It would be amazing if podcasts could have individual settings that if set override global settings.

### Steps to reproduce

When opening the podcast information screen, I imagine there could be a small square to the right of the description with the drop down menu for:
"Check for new episodes"
"Keep"
"When new episodes are available"

But with an additional checkbox called "Obey Global Settings" that is by default checked which makes the options above greyed out, but when unchecked allows you to set specific options for that podcast. 

## System information

 * **Airsonic version**: 10.6.2-RELEASE – May 3, 2020   /  Apache Tomcat/8.5.51, java 1.8.0_242, Linux (174.3 MB / 253.5 MB)
 * **Operating system**: Docker on TOS 5.0.156

**Comment by dsiminiuk** [comment]
In 11.0.0-SNAPSHOT.20220625052932 there are individual download and retention settings for individual podcasts as well as a default. Time for you to upgrade maybe?

**Comment by judicandus** [comment]
> In 11.0.0-SNAPSHOT.20220625052932 there are individual download and retention settings for individual podcasts as well as a default. Time for you to upgrade maybe?

Hi Danny, thank you for your answer. 

Could you please explain how to do the upgrade? I just installed it last week using the instructions for installation referenced in the README file for the Docker installation (https://hub.docker.com/r/airsonicadvanced/airsonic-advanced) . I installed it using the latest docker hub package so I thought I was using the latest version. 

Thank you,

**Comment by dsiminiuk** [comment]
@judicandus I am definitely not the person to ask about upgrading. I am an old legacy Airsonic user with a worthless paid license that I abandoned in favor of this Advanced project where things are actually under active development. I've also been ignoring Docker because I have no use for it in my own use case.

 I started over from scratch on a clean Ubuntu VM and it's been great. Hopefully some others here can give you a leg up on an upgrade.
